tinygrad/helpers.py

```diff
@@ -27,7 +27,12 @@ def __call__(self, x): self.value = x
   def __bool__(self): return self.value != 0
   def __ge__(self, x): return self.value >= x
 
-DEBUG, IMAGE = DebugSingleton(), getenv("IMAGE", 0)
+class HalfSingleton:
```

Comment:
Definitely not.

Reply:
the class, env var or both?

Reply:
Both. If you want this sort of stuff, it has to follow normal broadcasting rules.